hermes-console-vue/src/store/consistency/useConsistencyStore.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { beforeEach, describe, expect } from 'vitest';
+import { createPinia, setActivePinia } from 'pinia';
+import { dummyGroupInconsistency } from '@/dummy/groupInconsistency';
+import { dummyInconsistentGroups } from '@/dummy/inconsistentGroups';
+import {
+  fetchConsistencyGroupsHandler,
+  fetchGroupInconsistenciesHandler,
+} from '@/mocks/handlers';
+import { setupServer } from 'msw/node';
+import { useConsistencyStore } from '@/store/consistency/useConsistencyStore';
+
+describe('useConsistencyStore', () => {
+  const server = setupServer();
+
+  beforeEach(() => {
+    setActivePinia(createPinia());
+  });
+
+  it('should initialize', async () => {
+    // when
+    const consistencyStore = useConsistencyStore();
+
+    // then
+    expect(consistencyStore.groups).toEqual([]);
+    expect(consistencyStore.progressPercent).toEqual(0);
+    expect(consistencyStore.fetchInProgress).toBeFalsy();
+  });
+
+  it('should fetch group consistency', async () => {
```

Comment:
please add test where server return e.g. 500 status code and check if 'consistencyStore.error.fetchError' is present

Reply:
tests added